README.md

```diff
@@ -1,22 +1,24 @@
-# iOS/macOS Swift SDK for LiveKit
+# Swift Multiplatform (iOS and macOS) SDK for LiveKit
 
-Official Client SDK for [LiveKit](https://github.com/livekit/livekit-server).
-Easily add video & audio capabilities to your iOS/macOS apps.
+This is the official LiveKit Swift client SDK. 
 
-## Docs & Example app
+Together with LiveKit [server](https://github.com/livekit/livekit-server), you can easily add real-time video/audio/data capabilities to your iOS or macOS apps.
 
-Docs and guides are at [https://docs.livekit.io](https://docs.livekit.io).
+<insert gif/image here />
 
-There is full source code of a [iOS/macOS Swift UI Example App](https://github.com/livekit/client-example-swift).
-
-For minimal examples view this repo 👉 [Swift SDK Examples](https://github.com/livekit/client-example-collection-swift)
+## Features
 
 ## Installation
 
-LiveKit for Swift is available as a Swift Package.
+### Requirements
+
+### Swift Package Manager
```

Comment:
we now support Cocoapods too, would be great to document steps here.

Reply:
Oh, nice, yeah, let's do that.